eunkyung/services/userService.js

```diff
@@ -0,0 +1,37 @@
+const userDao = require("../models/userDao");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const createUser = async (name, password, email, profileImage) => {
+  const saltRounds = 12;
+  const hashedPw = await bcrypt.hash(password, saltRounds);
+
+  const createUserResult = await userDao.createUser(
+    name,
+    hashedPw,
+    email,
+    profileImage
+  );
+  return createUserResult;
+};
+
+const emailCheckLogin = async (email, password) => {
+  const [userInfo] = await userDao.emailCheckLogin(email);
+  if (!userInfo) {
+    return undefined;
+  }
```

Comment:
로그인을 할 때 유저가 보낸 데이터와 일치하는 값이 없다면 undefined가 아니라 회원가입을 할 수 있도록 에러메시지를 반환해주어야 할 것 같습니다.
```suggestion
  if (!userInfo) {
    throw new Error("Invalid User");
  }
```